src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'underscore';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { SidebarView } from 'js/views/patients/patient/sidebar/sidebar_views';
+
+export default App.extend({
+  onBeforeStart({ patient }) {
+    this.patient = patient;
+
+    this.showView(new SidebarView({ model: this.patient }));
```

Comment:
I think this app should probably be 
```
onBeforeStart() {
...
  this.showView...
  this.showChildView('engagement', new Preloader());
},
onStart() {
  this.showChildView('engagement', new EngagementView({ model: this.patient });
},
onFail() {
  this.showChildView('engagement', new NotAvailableView());
},
```

Reply:
Actually I think onFail can also show EngagementView and we should show not available `{{#unless engagement.status}}`